### ISSUE TITLE
New Rule: no-missing-generic-family-keyword

### ISSUE THREAD

**Issue by TKDev7** [eslint/css#155]
### Rule details

Disallow a missing generic family keyword within font families.

### What type of rule is this?

Warns about a potential problem

### Example code

```css
a { font-family: Arial, sans-serif; }
/**                     ↑
 * An example of generic family name */
```

### Participation

- [x] I am willing to submit a pull request to implement this rule.

### Additional comments

From [CSS Fonts Module](https://drafts.csswg.org/css-fonts/#font-family-prop):

> Authors are encouraged to append a generic font family as a last alternative for improved robustness, and to use a more specific generic font family, if applicable, to prefer a specific style even if the individual named font family is not available.

**Comment by nzakas** [comment]
Thanks for the suggestion. I think we should find a way to merge this concept with #153
